internal/charmstore/search.go

```diff
@@ -347,6 +356,8 @@ type SearchParams struct {
 	Include []string
 	// Start the the returned items at a specific offset.
 	Skip int
+	// Additional ACL groups to search
```

Comment:
This is really users or group right?
If so, could you specify that in the description? Also please complete the sentence.